src/vs/workbench/contrib/terminal/browser/remoteTerminalBackend.ts

```diff
@@ -260,6 +261,19 @@ class RemoteTerminalBackend extends BaseTerminalBackend implements ITerminalBack
 		return this._remoteTerminalChannel.listProcesses();
 	}
 
+	async getLatency(): Promise<IPtyHostLatencyMeasurement[]> {
+		const sw = new StopWatch();
+		const results = await this._remoteTerminalChannel.getLatency();
+		sw.stop();
+		return [
+			{
+				label: 'window<->ptyhostservice<->ptyhost',
```

Comment:
should this label include `remote`?

Reply:
oh nvm looks like that happens https://github.com/microsoft/vscode/pull/187281/files#diff-38cb820e08ef9a8479ff998378fa4b33cd3a9e57461329ded57f8a4b5e8dae4aR396